api/views.py

```diff
@@ -67,10 +67,13 @@ def create(self, request, **kwargs):
                 answer.status = Answer.CORRECT
                 puzzle.answer = answer.text
                 if puzzle.chat_room:
-                    puzzle.chat_room.archive_channels()
-                    msg = f"{puzzle.name} has been solved with {answer.text}!"
-                    puzzle.chat_room.get_service().announce(msg)
-                    puzzle.chat_room.send_message(msg)
+                    try:
+                        puzzle.chat_room.archive_channels()
+                        msg = f"{puzzle.name} has been solved with {answer.text}!"
+                        puzzle.chat_room.get_service().announce(msg)
+                        puzzle.chat_room.send_message(msg)
+                    except:
+                        logger.warn("Chat operations failed.")
```

Comment:
Can we include the exception in the log warning? Otherwise, I think it might eat the error message here.

```
except Exception as e:
    logger.warn("Chat operations failed: %s", e)
````